### PR TITLE
Support PUT requests

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -37,6 +38,7 @@
         "phpstan/phpstan-phpunit": "^0.11",
         "phpunit/phpunit": "^8.0",
         "symfony/debug": "^4.2",
+        "symfony/stopwatch": "^4.2",
```
Comment (thewilkybarkid) [inline composer.json:45]: Unrelated, but adds more to the profiler.

File: tests/ContentServiceTest.php [hunk 2]
```diff
@@ -83,4 +83,37 @@ public function it_gets_an_item() : void
...
+    </meta>
+</item>
+XML
+        );
+
+        $response = $this->handle($request);
+
+        $this->assertSame(Response::HTTP_NO_CONTENT, $response->getStatusCode());
+
+        $response = $this->handle(Request::create('/items/new-article/versions/1'));
```
Comment (thewilkybarkid) [inline tests/ContentServiceTest.php:104]: This is actually being written to the `tests/fixtures` directory.

**Comment by giorgiosironi** [comment]
Most of what's actually happening here is in https://github.com/libero/content-api-bundle/pull/22

File: tests/ContentServiceTest.php [hunk 3]
```diff
@@ -83,4 +83,37 @@ public function it_gets_an_item() : void
         $this->assertSame('application/xml; charset=utf-8', $response->headers->get('Content-Type'));
         $this->assertXmlStringEqualsXmlFile(self::ARTICLES_PATH.'/article1/1.xml', $response->getContent());
     }
+
+    /**
```
Comment (giorgiosironi) [inline tests/ContentServiceTest.php:76]: The specification from the test is clear

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**